CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v3.0.8
+
+### Bug fixes
+
+* [4691](https://github.com/api-platform/core/pull/4691) Symfony: Fix swagger config to support Symfony ConfigBuilders  (#4691)
```

Comment:
@soyuka isn't it auto-generated, or maybe do you have another process for it?

Reply:
yes it is auto generated please remove theses changes

Reply:
Done